other/edge-server/client/README.md

```diff
@@ -9,15 +9,15 @@ React client -> Express server (Node.js Driver) -> Edge Server -> Atlas
 
 To use this example application, you must:
 
-1. Create an App Services App based on the `edge-server.todo` template app, and
-   get Edge Server enabled for your new app.
+1. Create an App Services App based on the `edge-server.todo` template app.
```

Comment:
With these steps here, _plus_ the quick start below, and then additional details as part of the quick start, it feels like we're providing three sets of instructions. I think I would just remove these steps, move the "Project npm commands" section down, and go directly into the quick start after the high-level summary stuff that is currently ln 17- ln 30.

Reply:
Agreed. It was definitely confusing before. I've removed that first set of instructions.